### PR TITLE
ICU-20198 Add issue browse link to PR template.

### PR DESCRIPTION
**Pull request by sffc**
This change will create a link from GitHub to Jira on every PR.  Right now a browser extension is required for this functionality.

The issue is closed.  I can open a new issue or re-open the old one to accept a new commit.

##### Checklist

- [x] Issue filed at https://unicode-org.atlassian.net :  ICU-20003
- [x] Update PR title to include Issue number
- [ ] Issue accepted
- [ ] Tests included
- [ ] Documentation is changed or added

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ You will be automatically asked to sign the contributors license before the PR i
 
 ##### Checklist
 
-- [ ] Issue filed at https://unicode-org.atlassian.net :  ICU-______
-- [ ] Update PR title to include Issue number
+- [ ] Issue filed: https://unicode-org.atlassian.net/browse/ICU-_____
```
Comment (srl295) [inline .github/pull_request_template.md]: the point is for people to create a new ticket. https://unicode-org.atlassian.net/browse/ICU-_____ is a dead link.

Comment (srl295) [reply .github/pull_request_template.md]: I guess you can still create a ticket from there. I'll approve.

Comment (jefgen) [reply .github/pull_request_template.md]: I suppose we might have a direct link as well?
Maybe add another step?
- [ ] Issue filed. [Create Issue](https://unicode-org.atlassian.net/secure/CreateIssue!default.jspa?selectedProjectId=1)
- [ ] Link to issue: https://unicode-org.atlassian.net/browse/ICU-_____
- [ ] ...etc..

Comment (sffc) [reply .github/pull_request_template.md]: I think it's obvious enough where to go to file an issue.  Plus, the "Create Issue" link doesn't work unless you look at the Markdown preview.

**Pull request merged**